.github/workflows/supabase-types.yml

```diff
@@ -0,0 +1,47 @@
+name: Update database types
+
+on:
+  push:
```

Comment:
I think this should happen only on the main branch, so it's the latest types and it doesn't bombard the repo with commits.